### ISSUE TITLE
Questions on GenerativeModel() training process and its parameters.

### ISSUE THREAD

**Issue by zegzag** [HazyResearch/snorkel#1092]
I've learned from the tutorials that the parameters of `GenerativeModel.train()` method can be only  with `L` matrix from LFs and I didn't find that the golden label  Y parameters is necessary in  `GenerativeModel.train()`.  

But when I read the paper of _"DataProgramming: Creating Large Training Sets, Quickly"_, I found that the model is trained to fit the joint distribution of (LFs, Y) ["_Formula: (1)_ in this paper" ] by maximum likelihood estimation method, which means that the true label of Y is also needed while training.   

So I am confused how is the model able to fit to the true distribution of golden label Y only with the `L` matrix and without Y.  And if I generate the `L` randomly (eg: `np.random.randint(0, 5, 1000).reshape(100, 10)` ( 5 categories, 10 LFs, 100 samples ) ) and only use this `L` to train the model,   what distribution the model is fitted with  in this case ?

**Comment by YipingNUS** [comment]
@zegzag, you can refer to the paper "Snorkel: Rapid Training Data Creation with Weak Supervision" section 2.2. 

> The core operation of Snorkel is modeling and integrat-
ing the noisy signals provided by a set of labeling func- tions. Using the recently proposed approach of data pro- gramming [5, 38], we model the true class label for a data point as a latent variable in a probabilistic model. In

**Comment by paroma** [comment]
Closing this issue for now, feel free to reopen!
